src/watchmaker/status/providers/abstract.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""Abstract Status Provider."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals, with_statement)
+
+import abc
+
+import six
+
+
+class StatusProviderError(Exception):
```

Comment:
Thoughts on putting this in the `watchmaker.exceptions` module?

Reply:
Makes sense.